src/configs/YITHPlugins.config.js

```diff
@@ -87,12 +87,12 @@ export const YITHPluginsDefinitions = (props) => ({
         Image: Search,
       }),
       text: ({ isActive }) => ({
-        title: __('YITH WooCommerce AJAX Search', 'wp-module-ecommerce'),
+        title: __("YITH WooCommerce AJAX Search", 'wp-module-ecommerce'),
         description: __(
-          'YITH WooCommerce AJAX Search is an effective and predictive real-time search engine on your store so users can quickly find products they want. Speed up your search and boost your sales.',
+          "YITH WooCommerce AJAX Search is an effective and predictive real-time search engine on your store so users can quickly find products they want. Speed up your search and boost your sales.",
           'wp-module-ecommerce'
         ),
-        actionName: isActive ? 'Manage' : 'Enable',
+        actionName: isActive ? __("Manage") : __("Enable"),
```

Comment:
The translation functions here should have a text domain assigned or they won't get translated. The strings should also use single-quotes.